src/style.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
+.card--width {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  color: #060b35;
+}
+
+.card__img {
+  display: block;
+  height: 134px;
+
+  width: 100%;
+  margin-bottom: 40px;
+  padding: 0 3px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+  margin-bottom: 16px;
+}
+
+.card__rating-block {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  content: url("./images/star.svg");
```

Comment:
Add star image as background (copy from stars task or check in our morning lesson)
Don't forget about background-position: center property
It should fix tests